refresh/templates/common/RouteTests.swift

```diff
@@ -58,8 +58,37 @@ class RouteTests: XCTestCase {
 
         waitForExpectations(timeout: 10.0, handler: nil)
     }
-
+    
+    func testHealthRoute() {
+        
+        let printExpectation = expectation(description: "The /health route will print UP, followed by a timestamp.")
+        
+        URLRequest(forTestWithMethod: "GET", route: "health")?
+            .sendForTestingWithKitura { data, statusCode in
+                if let getResult = String(data: data, encoding: String.Encoding.utf8) {
+                    XCTAssertEqual(statusCode, 200)
+                    XCTAssertTrue(getResult.contains("UP"))
+                    let date = Date()
+                    let calendar = Calendar.current
+                    let yearString = String(describing: calendar.component(.year, from: date))
+                    XCTAssertTrue(getResult.contains(yearString)
+                } else {
+                    XCTFail("Return value from /health was nil, or was not UP, or had no timestamp.")
```

Comment:
I believe this error message doesn't give correct information about the error. 
It is better to put the errors inside the asserts such as : 
`XCTAssertEqual(statusCode, 200, "Error")`
And set this XCTFail to "The conversion to String(..) failed" something along those lines which is what the `if ` is checking for.